new/kitchen_telemetry.md

```diff
@@ -0,0 +1,143 @@
+---
+RFC: unassigned
+Title: Use of telemetry in Test Kitchen
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Use of telemetry in Test Kitchen
+
+This RFC is a description of Test Kitchen's implementation of <the
+telemetry RFC, as yet unumbered>, as required in that RFC.
+
+## What questions do we hope to answer?
+
+ * What platforms do our users run kitchen on?
+ * Which plugins are used by kitchen users?
+ * Which bento boxes are used?
+ * Which provisioners are used?
+ * If a chef provisioner is used, which chef versions are installed?
+ * Which commands are used?
+
+## Implementation
+
+The bulk of this RFC is the description of the events, below. We
+envisage that a single client library will be built for all chef
+projects, and Kitchen will consume that to send events. That library
+would ensure events are sent in a non-blocking manner, and would deal
+with retries, session handling and opt-out.
```

Comment:
How would you deal with a standalone client, which may never be connected to the internet? Is there a throttling or exponential backoff process which would cause the system to never try reporting again even though the user may not have configured the anonymous data collection configuration file (i.e. PR #269) to prevent data collection? Please have a way to keep this in check and even disabled automatically in unexpected circumstances.